Bio/Blast/NCBIWWW.py

```diff
@@ -67,6 +68,11 @@ def qblast(program, database, sequence, url_base=NCBI_BLAST_URL,
      - hitlist_size   Number of hits to return. Default 50
      - megablast      TRUE/FALSE whether to use MEga BLAST algorithm (blastn only)
      - service        plain, psi, phi, rpsblast, megablast (lower case)
+     - organism       Specify an organism while using nr/nt database, or an SRA accession
+                      for SRA, EST and other databases. The organism also could be specified
+                      by "entrez_query". For example: "entrez_query='rat [ORGN]'" is identical
```

Comment:
Shouldn't that be:  ``"entrez_query='rat[ORGN]'"`` (no space before the opening square bracket)?

Reply:
It looks llike both `rat[ORGN]` (without space) and `rat [ORGN]` (with space) could get expected results after I tested. `rat [ORGN]` (with space) was used by the original version.